src/tutorial/runtime/introduction.rst

```diff
@@ -69,16 +69,16 @@ We can also call other scripts or executables in this way, e.g:
            script = ~/foo/bar/baz/hello_world
 
 
-:envvar:`PATH` and :envvar:`PYTHONPATH`
----------------------------------------
+``PATH`` and :envvar:`PYTHONPATH`
```

Comment:
The Python reference material does not document the `PATH` variable and I can't find anywhere appropriate to source it from so the simple solution is to demote it from a reference to a string literal.